electron-app/src/server/websites/cohost/cohost.controller.ts

```diff
@@ -0,0 +1,10 @@
+import { Controller } from '@nestjs/common';
+import { GenericWebsiteController } from '../generic/generic.controller';
+import { Cohost } from './cohost.service';
+
+@Controller('cohost')
+export class CohostController extends GenericWebsiteController {
```

Comment:
suggestion: I think the controller might be removable.
The controllers for specific websites are usually created when something in the UI needs to get info from the service. Typically done with websites that have folders.